packages/server/src/config/types.ts

```diff
@@ -74,6 +74,9 @@ export interface MedplumServerConfig {
   /** Max length of Bot AuditEvent.outcomeDesc when logging to logger */
   maxBotLogLengthForLogs?: number;
 
+  /** Search strategy system repositories use when using token search parameters. */
+  systemRepositoryTokenReadStrategy?: 'unified-tokens-column' | 'token-tables';
```

Comment:
👍 

This setting should go away after token-tables are fully deprecated, correct?

Reply:
Correct!